charon/Cargo.toml

```diff
@@ -55,9 +55,6 @@ walkdir = "2.3.2"
 which = "6.0.1"
 
 hax-frontend-exporter = { git = "https://github.com/hacspec/hax", branch = "main" }
-hax-frontend-exporter-options = { git = "https://github.com/hacspec/hax", branch = "main" }
-# hax-frontend-exporter = { path = "../../hax/frontend/exporter" }
```

Comment:
Can you keep this one? I use it all the time to use my local hax instead of the locked one

Reply:
Oh, sorry, yes sure! :)